Dockerfile

```diff
@@ -49,10 +49,11 @@ RUN sudo pip3 install --break-system-packages .
 
 RUN sudo python3 prerequisites.py --yes --cmsuser=cmsuser install
 
-RUN sudo sed 's|/cmsuser:your_password_here@localhost:5432/cmsdb"|/postgres@testdb:5432/cmsdbfortesting"|' ./config/cms.conf.sample \
+RUN sudo sed -e 's|/cmsuser:your_password_here@localhost:5432/cmsdb"|/postgres@testdb:5432/cmsdbfortesting"|' ./config/cms.conf.sample \
```

Comment:
nit: we can probably remove the first sudo from these 3 lines? (I think we only need it for `tee`, not for `sed`)